### PR TITLE
[1LP][RFR] Added paginator to ContainerObjectAllBaseView + fixed test_tables_sort

### PR DESCRIPTION
**Pull request by gshefer**
- Changed ContainerObjectAllBaseView to inherit ProvidersView.
- fixed test_tables_sort

{{ pytest: cfme/tests/containers/test_tables_sort.py --use-provider cm-env2 }}

### CODE REVIEW

**Comment by gshefer** [comment]
@izapolsk Cool, I modified accordingly hope it's better now.

File: cfme/common/provider_views.py [hunk 1]
```diff
@@ -313,6 +314,7 @@ class ContainersProvidersView(ProvidersView):
      represents Main view displaying all Containers providers
     """
     table = Table(locator="//div[@id='list_grid']//table")
+    paginator = View.nested(PaginationPane)
```
Comment (izapolsk) [inline cfme/common/provider_views.py]: EntitiesView which is used in ProvidersView is a bit sophisticated. So, you probably didn't notice that paginator is already there. Nevertheless,  paginator is already present there. You don't need to add a new one.

https://github.com/izapolsk/integration_tests/blob/master/widgetastic_manageiq/__init__.py#L1980

File: cfme/common/provider_views.py [hunk 2]
```diff
@@ -17,7 +17,8 @@
                                   BaseEntitiesView,
                                   DynaTree,
                                   BootstrapTreeview,
-                                  ProviderEntity)
+                                  ProviderEntity,
+                                  PaginationPane)
```
Comment (izapolsk) [inline cfme/common/provider_views.py]: you don't need this import

**Comment by gshefer** [comment]
@izapolsk Done.

**Pull request merged**